db/migrate/20240310003914_migrate_facecasts.rb

```diff
@@ -0,0 +1,21 @@
+class MigrateFacecasts < ActiveRecord::Migration[7.1]
+  def up
+    Character.where.not(pb: nil).in_batches do |batch|
+      ary = batch.pluck(:id, :pb, :user_id)
+      ary.each do |id, pb, user_id|
+        galleries = GalleryTag.where(character_id: id).pluck(:gallery_id)
+        fcs = pb.split(/[,;] (?=[A-Z])|\//).map(&:strip)
+        fcs.each do |fc|
+          tag = Facecast.find_by(name: fc)
+          tag = Facecast.create!(name: fc, user_id: user_id) if tag.nil?
+          CharacterTag.create!(character_id: id, tag_id: tag.id)
+          galleries.each { |gid| GalleryTag.create!(gallery_id: gid, tag_id: id) }
+        end
+      end
+    end
+  end
+
+  def down
+    Facecast.each(&:destroy!)
```

Comment:
We're not deleting the `pb` field on Character with this change, then? I think it might be nice to delete the column for cleanup, but if we do that, we need to restore it in `def down`, too, in case we need to revert things in prod 🤔

Reply:
...oh, hm, yeah, if we're doing this as a migration rather than a migration script (which we might not want to!) the logical thing would be to remove that in the migration too